server/utils.go

```diff
@@ -19,13 +19,24 @@ package server
 import (
```

Comment:
i think azureutil.go is more fitting than utils.go

Reply:
shifted functions to util/azure/utils.go